source/examples/header-contents.rst

```diff
@@ -0,0 +1,52 @@
+Understand the contents of the header
+*************************************
+
+Problem
+=======
+
+The databroker provides the metadata in a dictionary of names and values that
+we call the *header*. Its organized but rather sprawling contents can be
+difficult to navigate.
+
+Approach
+========
+
+Here, we explain the overall structure, and we highlight some commonly useful
+contents and where to find them.
+
+Structure
+---------
+
+A given header ``h`` has three main pieces:
+
+* the 'start' document, ``h['start']``, contains most of the interesting
+  metadata --- everything we know before the run starts. (What kind of scan
+  are we doing? Why? Who? When?)
+* the 'stop' document, ``h['stop']``, contains what we only know when
+  the scan is complete --- mainly, did it terminate successfully?
+* a list of 'descriptor' documents in ``h['descriptors']`` which serve
+  a sort of table of contents, giving metadata about each field (PV name,
```

Comment:
`s/about each field/about each data field in the corresponding event document/g`